src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/index.js

```diff
@@ -0,0 +1,20 @@
+// Node modules.
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default (store, widgetType) => {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+  if (root) {
+    import(/* webpackChunkName: "App" */
+    './components/App').then(module => {
+      const App = module.default;
+      ReactDOM.render(
+        <Provider store={store}>
+          <App />
+        </Provider>,
```

Comment:
We talked about this over Slack but I'm still curious if `<App store={store}/>` works the same way as wrapping in a `Provider`

Reply:
I think that would work but only if the root element of `App` were a `Provider`, right? I can move that Provider logic into App if you think that'd be cleaner! 🙂

Reply:
Nah it's cool the way it is. For some reason other widgets are able to get away without using a Provider 🤷